LDAR_Sim/external_sensors/METEC_WIND.py

```diff
@@ -0,0 +1,154 @@
+                site_true_rate (float): same as site_rate
+                site_measured_rate (float): total emis from all leaks measured
+                equip_measured_rates (list): total of all leaks measured for each equip group
+                venting (float): same as input
+                found_leak (boolean): Did the crew find at least one leak at the site
+    """
+    missed_leaks_str = '{}_missed_leaks'.format(self.config['label'])
+    equip_measured_rates = []
+    site_measured_rate = 0
+    found_leak = False
+    n_leaks = len(covered_leaks)
+    mdl = self.config['sensor']['MDL']
+
+    hourly_weather = self.state['weather'].get_hourly_weather(
+        ['winds'],
+        self.state['t'].current_date,
+        number_of_hours=1,
+        site=site)
+    average_wind = np.average(hourly_weather['winds']) * 3.6  # m/s to km/hr
+
+    if self.config["measurement_scale"] == "site":
+        # factor of 3.6 converts g/s to kg/h
+        rate_per_wind = covered_site_rate * 3.6 / average_wind
+        prob_detect = 1/(1+np.exp(mdl[0]-mdl[1]*rate_per_wind))
+        if rate_per_wind <= (mdl[2]*3.6):
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+        elif np.random.binomial(1, prob_detect):
+            found_leak = True
+            site_measured_rate = measured_rate(
+                covered_site_rate, self.config['sensor']['QE'])
+        else:
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+    elif self.config["measurement_scale"] == "equipment":
+        for rate in covered_equipment_rates:
+            m_rate = measured_rate(rate, self.config['sensor']['QE'])
+            rate_per_wind = m_rate * 3.6 / average_wind
+            prob_detect = 1 / \
+                (1+np.exp(mdl[0]-mdl[1]*m_rate))
+            if rate_per_wind <= (mdl[2]*3.6):
+                m_rate = 0
+            elif np.random.binomial(1, prob_detect):
+                found_leak = True
+            else:
+                m_rate = 0
+            equip_measured_rates.append(m_rate)
+            site_measured_rate += m_rate
+        if not found_leak:
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+    elif self.config['measurement_scale'] == 'component':
+        for leak in covered_leaks:
+            rate_per_wind = leak['rate'] * 3.6 / average_wind
+            prob_detect = 1 / \
+                (1+np.exp(mdl[0]-mdl[1]*rate_per_wind))
+            if prob_detect >= 1:
+                prob_detect = 1
+            if rate_per_wind <= (mdl[2]*3.6):
+                site[missed_leaks_str] += 1
+                self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += 1
+            elif np.random.binomial(1, prob_detect):
+                found_leak = True
```

Comment:
Question: What is this boolean being used for in this section of the if statement?